modules/apps/list-type/list-type-test/src/testIntegration/java/com/liferay/list/type/service/test/ListTypeEntryLocalServiceTest.java

```diff
@@ -113,153 +124,139 @@ public void testAddListTypeEntry() throws Exception {
 				listTypeEntryKeyException.getMessage());
 		}
 
-		ListTypeEntry listTypeEntry =
-			ListTypeEntryLocalServiceUtil.addListTypeEntry(
-				null, TestPropsValues.getUserId(),
-				_listTypeDefinition.getListTypeDefinitionId(), "able",
-				Collections.singletonMap(
-					LocaleUtil.US, RandomTestUtil.randomString()));
-
 		Assert.assertEquals(
```

Comment:
I think we don't need to check that, because it's filled by a generated code. I didn't find it in another service test on objects module.